db/migrate/20250210184018_add_jobs_finished_at_to_good_job_batches.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddJobsFinishedAtToGoodJobBatches < ActiveRecord::Migration[7.1]
+  def change
+    reversible do |dir|
+      dir.up do
+        # Ensure this incremental update migration is idempotent
+        # with monolithic install migration.
+        return if connection.column_exists?(:good_job_batches, :jobs_finished_at)
+      end
+    end
+
+    change_table :good_job_batches do |t|
+      t.datetime :jobs_finished_at
+    end
```

Comment:
I would need to know the reason why this added to judge if it is necessary to add this migration.

Reply:
Same case as above, but it didn't have a explicit field for finish time.